src/JavaDependenciesManager.ts

```diff
@@ -74,20 +73,13 @@ export async function downloadSpecificCamelKJavaDependencies(
 		uri = vscode.window.activeTextEditor?.document.uri;
 	}
 	if (uri) {
-		const kamelLocal = kamelCli.create();
-		const parentDestination = parentDestinationFolderForDependencies(context);
-		await clearDestinationFolder(mainOutputChannel, parentDestination);
-		const command = `local build --integration-directory "${parentDestination}" "${uri.path}"`;
+		const destination = destinationFolderForDependencies(context);
+		await clearDestinationFolder(mainOutputChannel, destination);
+		const command = `jbang camel@apache/camel dependency copy --output-directory="${destination}" "${uri.path}"`;
 		try {
-			await kamelLocal.invoke(command);
+			execSync(command);
```

Comment:
when moving to VS Code Language support, it might be nicer to use a VS Code task so that user have access to log in case of problem